src/app/layouts/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -37,11 +56,15 @@ export const NavItem = ({ label, href, dropdownInner }: NavItemProps) => {
         onMouseEnter={onOpen}
         end
       >
-        <p className={cx('label')}>{label}</p>
+        <span className={cx('label')}>{label}</span>
         {dropdownInner && (
-          <span className={cx('dropdown-arrow')}>
+          <button
+            onKeyDown={handleKeyPress}
+            className={cx('dropdown-arrow')}
+            aria-expanded={isDropdownOpen}
+          >
             <DropdownArrow />
```

Comment:
When I focus on the `DropdownArrow` and press `Space`, it doesn't open the menu but changes the url

Reply:
I made it based on the article from related issue, where they said that users navigate with tab and select with enter. Not a big deal to handle some additional cases, will implement it.

Reply:
done